frontend/src/data_context_global.tsx

```diff
@@ -59,6 +59,9 @@ export interface Individual extends LimitedIndividual{
     alive: boolean
 }
 
+export type PrivacyLevel = 'private' | 'authenticated' | 'public' | null;
```

Comment:
I would change `null` to a string describing what privacy level it stands for. I don't know what `null` is here :)

Reply:
null is quite literally when the value is NULL in the database.